packages/opentelemetry-instrumentation-grpc/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@opentelemetry/instrumentation-grpc",
```

Comment:
Could you please use plugin instead of instrumentation https://github.com/open-telemetry/opentelemetry-js/pull/126#discussion_r306964711?. Plugin Loader will load packages with name starting with "**@opentelemetry/plugin-xxx**" See https://github.com/open-telemetry/opentelemetry-js/pull/126/files#diff-22ee4a6b410955eb761939a67be00d8eR29

Reply:
Thanks for notifying this. I didn't catch this one.
Done!